added some cites here and there (#9)
MrLongbottom/P9-Rapport: sections/Pipeline.tex

	\item body
\end{itemize}
When training the LDA model, we concatenate the \emph{headline} and \emph{body} for simplicity.

> The preprocessing phase is described in further detail in \ref{subsec:prepro}.

Reviewer: ```suggestion
The preprocessing phase is described in further detail in \autoref{subsec:prepro}.
```

This phase applies different \gls{NLP} methods to simplify the dataset and remove redundant information.
After the preprocessing phase, we are left with $\sim$130.000 articles.

\subsection{LDA}
We apply the \acrfull{lda} to the dataset to generate topic clusters based on the content of the articles. 

> We describe the investigation and selection of hyper parameters in \ref{subsec:lda}. 

Reviewer: ```suggestion
We describe the investigation and selection of hyper parameters in \autoref{subsec:lda}. 
```

After the model has been trained, we have a document-topic distribution matrix $\theta$ and a topic-word distribution matrix $\beta$.
$\theta$ will be used as clusters in the next phase.

\subsection{Cluster-based PageRank}

> In this phase, we apply cluster-based random walk, described in section \ref{sec:cluster_pagerank}, to the articles and clusters given in the \gls{lda} phase.

Reviewer: ```suggestion
In this phase, we apply cluster-based random walk, described in \autoref{sec:cluster_pagerank}, to the articles and clusters given in the \gls{lda} phase.
```
